File: isPalindrome.py
def isPalindrome(x: int) -> bool:
    x = str(x)
    l = []
    for i in range(len(x)):
        if x[i] == x[-i-1]:
            l.append(1)
        else:
            l.append(0)
    if min(l) == 1:
        return True
    else:
        return False

if __name__ == "__main__":
    print(isPalindrome(121))
    print(isPalindrome(11231234))

File: isPalindrome.cpp
#include<iostream>
#include<string>

bool isPalindrome(int x) {
        std::string s = std::to_string(x);
        int l = s.length();
        int end = l-1;
        for(int i = 0; i < l; i++){
            if(s[i] != s[end - i])
                return false;
        };
        return true;
    } 

int main(){
    int x = 121;
    std::cout << isPalindrome(x) << std::endl;
    return 0;
}